pkg/kn/commands/revision/revision_list_test.go

```diff
@@ -92,6 +92,23 @@ func TestRevisionListDefaultOutput(t *testing.T) {
 	assert.Check(t, util.ContainsAll(output[2], "bar-wxyz", "bar"))
 }
 
+func TestRevisionListDefaultOutputNoHeaders(t *testing.T) {
+	revision1 := createMockRevisionWithParams("foo-abcd", "foo")
+	revision2 := createMockRevisionWithParams("bar-wxyz", "bar")
+	RevisionList := &v1alpha1.RevisionList{Items: []v1alpha1.Revision{*revision1, *revision2}}
+	action, output, err := fakeRevisionList([]string{"revision", "list", "--no-headers"}, RevisionList)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if action == nil {
+		t.Errorf("No action")
+	} else if !action.Matches("list", "revisions") {
+		t.Errorf("Bad action %v", action)
+	}
+	assert.Check(t, util.ContainsAll(output[0], "foo-abcd", "foo"))
```

Comment:
Hmm, we are sending the output split by new lines, so the first line check against actual revision info sounds okay, but IMO we should also check that output doesnt have any headers listing.

@rhuss WDYT ?
One of thing I was wondering is if we should stop sending the output string split by new lines and should send the address of the whole output string (not separated by newlines), then individual tests can perform operation and check on it as per need.

Reply:
It's a good idea to test that no headers are included. However, as headers are in the first line, I would split up still in multiple lines, and check that the first line doesn't contain any header (like "SERVICE" or "NAME", or so).
Then all other lines should still contain the revision name (all lines, but probably there is only one anyway.) At the end I don't think its mission critical how we check here :-)